src/System.Drawing.Common/tests/mono/System.Drawing.Imaging/TiffCodecTests.cs

```diff
@@ -0,0 +1,323 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+//
+// TIFF Codec class testing unit
+//
+// Authors:
+//	Jordi Mas i Hernàndez (jordi@ximian.com)
+//	Sebastien Pouliot  <sebastien@ximian.com>
+//
+// Copyright (C) 2006, 2007 Novell, Inc (http://www.novell.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+using System;
+using System.Drawing;
+using System.Drawing.Imaging;
+using System.IO;
+using System.Security.Permissions;
+using System.Text;
+using Xunit;
+
+namespace MonoTests.System.Drawing.Imaging
+{
+
+    public class TiffCodecTest
+    {
+        /* Get suffix to add to the filename */
+        internal string getOutSufix()
+        {
+            string s;
+
+            int p = (int)Environment.OSVersion.Platform;
+            if ((p == 4) || (p == 128) || (p == 6))
+                s = "-unix";
+            else
+                s = "-windows";
+
+            if (Type.GetType("Mono.Runtime", false) == null)
+                s += "-msnet";
+            else
+                s += "-mono";
+
+            return s;
+        }
+
+        /* Checks bitmap features on a know 32bbp bitmap */
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void Bitmap32bitsFeatures()
+        {
+            string sInFile = Helpers.GetTestBitmapPath("almogaver32bits.tif");
+            using (Bitmap bmp = new Bitmap(sInFile))
+            {
+                GraphicsUnit unit = GraphicsUnit.World;
+                RectangleF rect = bmp.GetBounds(ref unit);
+                // MS reports 24 bpp while we report 32 bpp
```

Comment:
Ideally, this assertion would be in a separate test case, and disabled for Unix (against a descriptive issue). That way, the behavior difference is captured more succinctly and clearly than in a commented-out part of another test.